app/views/backend/product_nature_variants/show.html.haml

```diff
@@ -24,6 +24,31 @@
         %span.value
           %i.picto{class: "picto-stock"}
           = resource.current_stock.to_s + " " + resource.unit_name
+    - if resource.current_stock == 0
+      .med-info.important
+        %span.title CUMP
+        %span.value 0 €
```

Comment:
Why values are hard-coded? Use of localization is necessary here too. Use `.l` method to localize number or money.